helper.rb

```diff
@@ -0,0 +1,14 @@
+require "talk_like_a_pirate"
+
+
+def piratize(normal_sentence)
+  words_to_remove = ["gold", "treasure", "coin", "coins"]
+  arry_words = normal_sentence.split - words_to_remove
+  priate_talk = arry_words.join(" ")
+  return TalkLikeAPirate.translate(priate_talk)
+end
+
+def numberConverter(number)
+  top_num = number * 8
+  return top_num.round.to_s + "/8"
```

Comment:
The `return` is implied so there's no need to add an explicit one. Also the returned value appears to be a string. Have you checked out ruby's rational class? http://ruby-doc.org/core-2.2.0/Rational.html